Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN rm -rf \
   /code/SuperBuild/build/opencv \
   /code/SuperBuild/download \
   /code/SuperBuild/src/ceres \
-  /code/SuperBuild/src/entwine \
+  /code/SuperBuild/src/untwine \
```

Comment:
w00ooot!

Reply:
What does untwine give us vs entwine? It is entwine compatible from what I see, and HoBu, so good stuff, but I don't know enough about either library, apparently.

Reply:
It's the same app behind QGIS's new point cloud stuff. Faster, more memory efficient for individual machines.